pytorch_lightning/trainer/trainer.py

```diff
@@ -2473,6 +2474,33 @@ def _active_loop(self) -> Optional[Union[FitLoop, EvaluationLoop, PredictionLoop
     Logging properties
     """
 
+    @property
+    def logger(self) -> Optional[LightningLoggerBase]:
+        return self._logger
+
+    @logger.setter
+    def logger(self, new_logger: Optional[LightningLoggerBase]) -> None:
+        self.loggers = [new_logger] if new_logger else None
```

Comment:
```suggestion
        self.loggers = [new_logger] if new_logger else []
```

Didn't we say we want to make this an empty list?

Reply:
Yep, this gets handled when it is passed to the logger setter.

Reply:
I see. I think it would be a bit clearer to have this be an empty list, and the loggers setter should still work as intended